scipy/optimize/_minimize.py

```diff
@@ -38,6 +38,17 @@ def minimize(fun, x0, args=(), method=None, jac=None, hess=None,
              callback=None, options=None):
     """
     Minimization of scalar function of one or more variables.
+     
+    In general, the optimization problems are of the form:
+    
+    minimize f(x)
+    subect to: 
```

Comment:
Content: this is only about constraints, I'd mention this explicitly and also add bounds. Something like:

    minimize f(x), optionally within lower/upper bounds and satisfying constraints of the form::